.github/workflows/docker.build.yml

```diff
@@ -56,4 +56,19 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Create PR on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        with:
+          repository: rosalieper/wbaas-deploy
+      # Make changes to pull request here
```

Comment:
This needs to be a separate step from the checkout one above

Reply:
you are right, thanks